uiowa/uiowa
# [Bug] Layout builder edit bug

Resolves https://github.com/uiowa/uiowa/issues/8246



# How to test

```
ddev blt ds --site=its.uiowa.edu && ddev drush @its.local uli /admin/structure/types/manage/support_article/display/default/layout
```
1. Go to https://its.uiowa.ddev.siteadmin/structure/types/manage/support_article/display/default/layout and edit layout
3. Confirm that contextual block links are working to edit block
4. Go to https://its.uiowa.ddev.site/admin/structure/types/manage/service/display and confirm the same.
